tests/test_data_processing.py

```
@@ -1312,75 +1318,74 @@ def test_create_batch_dict_includes_relation_mappings(self, processor):
     def test_prepare_inputs_with_relations(self, processor):
         """Should add relation tokens to input."""
         texts = [["word1", "word2"]]
         entities = [["PER", "ORG"]]
         relations = [["WORKS_FOR"]]
-        
-        input_texts, prompt_lengths = processor.prepare_inputs(
-            texts, entities, blank=None, relations=relations
-        )
-        
+
+        input_texts, prompt_lengths = processor.prepare_inputs(texts, entities, blank=None, relations=relations)
```

Ruff (0.15.1)

[warning] Unpacked variable `prompt_lengths` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)